packages/dashboard/package.json

```
@@ -0,0 +1,80 @@
+{
+  "name": "@truffle/dashboard",
+  "version": "0.1.0",
+  "main": "dist/lib/index.js",
+  "types": "dist/lib/index.d.ts",
+  "files": [
+    "dist/lib",
+    "dist/bin"
+  ],
+  "dependencies": {
+    "@truffle/dashboard-message-bus": "^0.1.0",
+    "cors": "^2.8.5",
+    "debug": "^4.3.2",
+    "express": "^4.17.1",
+    "isomorphic-ws": "^4.0.1",
+    "open": "^8.4.0",
+    "ws": "^7.2.0"
+  },
+  "scripts": {
+    "start:react": "DISABLE_ESLINT_PLUGIN=true SKIP_PREFLIGHT_CHECK=true craco start",
+    "start:prebuilt": "node dist/bin/start-dev-server.js",
+    "start:dev-server": "ts-node --project tsconfig.server.json bin/start-dev-server.ts",
+    "start": "concurrently --kill-others \"yarn start:dev-server\" \"NODE_ENV=\"development\" yarn start:react\"",
+    "build:react": "DISABLE_ESLINT_PLUGIN=true SKIP_PREFLIGHT_CHECK=true craco build",
+    "build:server": "tsc -p tsconfig.server.json",
+    "copy:react": "cpy build/** dist/lib/dashboard-frontend/ --parents --dot",
+    "build": "yarn build:react && yarn build:server && yarn copy:react",
+    "prepare": "yarn build",
+    "test": "jest"
+  },
+  "browserslist": {
+    "production": [
+      ">0.2%",
+      "not dead",
+      "not op_mini all"
+    ],
+    "development": [
+      "last 1 chrome version",
+      "last 1 firefox version",
+      "last 1 safari version"
+    ]
+  },
+  "devDependencies": {
+    "@craco/craco": "^6.4.3",
+    "@testing-library/jest-dom": "^5.16.1",
+    "@testing-library/react": "^12.1.2",
+    "@testing-library/user-event": "^13.5.0",
+    "@types/cors": "^2.8.12",
+    "@types/ethereum-protocol": "^1.0.2",
+    "@types/express": "^4.17.13",
+    "@types/jest": "^27.0.3",
+    "@types/node": "^12.0.0",
+    "@types/react": "^17.0.37",
+    "@types/react-dom": "^17.0.11",
+    "@types/web3": "^1.2.2",
+    "@types/ws": "^7.2.0",
+    "@web3-react/core": "^6.1.9",
+    "@web3-react/injected-connector": "^6.0.7",
+    "autoprefixer": "^9",
+    "axios": "^0.24.0",
+    "concurrently": "^6.5.1",
+    "cpy-cli": "^3.1.1",
+    "delay": "^5.0.0",
+    "ethereum-protocol": "^1.0.1",
+    "ethers": "^5.5.2",
+    "jest": "^27.4.5",
+    "postcss": "^7",
+    "react": "^17.0.2",
+    "react-dom": "^17.0.2",
+    "react-json-view": "^1.21.3",
+    "react-scripts": "^4.0.0",
+    "tailwindcss": "npm:@tailwindcss/postcss7-compat",
+    "ts-jest": "^27.1.2",
+    "ts-node": "^10.4.0",
+    "typescript": "^4.1.4"
+  },
```

we'll need one of these (i learned the hard way)

```suggestion
  },
  "publishConfig": {
    "access": "public"
  },
```